imap_processing/cdf/config/imap_ultra_l1b_variable_attrs.yaml

```diff
@@ -236,6 +236,20 @@ elevation:
   LABLAXIS: elevation
   UNITS: radians
 
+phi:
+  <<: *default_float32
+  CATDESC: Ultra instrument frame event phi (azimuth with range -90 -> 90).
```

Comment:
-180 -> 180 ?

Reply:
For azimuth, I also thought we settled on [0, 360) rather than [-180, 180)?
Perhaps this was just a copy/paste issue from Elevation below?

Secondly, is it okay to have this in degrees, and az/el above it in radians?

Reply:
Ah, I think I now understand that this is fundementally separate from the other type of azimuth, and **only covers the half circle** in the direction of the slit, from the point of view of the detector. Is that why this would be -90 to 90 degrees?